src/expandable-block/block.scss

```
@@ -167,4 +167,35 @@
       margin-left: $iui-s;
     }
   }
+
+  &.iui-borderless {
+    @include themed {
+      background-color: transparent;
+      border: initial;
+    }
+
+    .iui-header {
+      @include themed {
+        background-color: transparent;
+        border: initial;
+      }
```

you don't need to use the `themed` mixin if you're not using any of our variables.

```suggestion
      background-color: transparent;
      border: initial;
```

Same thing in other places that you are including this mixin

---

removed